org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/dom/ASTConverterMarkdownTest.java

```diff
@@ -1839,4 +1839,23 @@ public class ArrayInCode {
 			}
 		}
 	}
+
+	public void testIllegelTagElement() throws JavaModelException {
```

Comment:
Not sure if this test is sufficient. This passes even when I rollback the changes to AbstractCommentParser.